<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: #f7f2ea;
color: #7a0606;
font-family: VCR,Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace;
}

QPushButton{
background-color: #cbbeb5;
}

</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>20</y>
      <width>271</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Log In</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>VCR,Courier New,Courier,Lucida Sans Typewriter,Lucida Typewriter,monospace</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">text-align:center;
</string>
       </property>
       <property name="text">
        <string>&lt;p style: &quot;text-align:center; &quot;&gt;SMARTPHONE&lt;br&gt;ONLINE&lt;br&gt;STORE&lt;/p&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>      ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_user"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_pass">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="push_login">
         <property name="text">
          <string>Iniciar Sesi√≥n</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="push_registro">
         <property name="text">
          <string>Registro</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/celulares/img/celulares/iPhoneXR.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stateLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionNuevo_Cliente"/>
    <addaction name="separator"/>
    <addaction name="actionSobre_Qt"/>
   </widget>
   <addaction name="menuOpciones"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevo_Cliente"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
  </widget>
  <action name="actionSobre_Qt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/img/png/007-package.png</normaloff>:/Icons/img/png/007-package.png</iconset>
   </property>
   <property name="text">
    <string>Sobre Qt</string>
   </property>
  </action>
  <action name="actionNuevo_Cliente">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/Icons/img/png/004-businessman.png</normaloff>:/Icons/img/png/004-businessman.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Cliente</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
